.github/workflows/release.yml

```
@@ -8,22 +8,25 @@ on:
         type: boolean
         default: false
         description: "This release will be labeled as non-production ready"
-  push:
+  pull_request:
+    types:
+      - closed
     branches:
-      - develop
+      - main
 
 jobs:
-  release:
+  candidate:
+    if: github.event_name == 'workflow_dispatch' || (github.event.pull_request.merged && !startsWith(github.head_ref, 'promote/'))
     permissions:
       contents: write
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
       - name: Prepare the next main release
-        uses: Adyen/release-automation-action@v1.0.0
+        uses: Adyen/release-automation-action@v1.1.1
         with:
           token: ${{ secrets.ADYEN_AUTOMATION_BOT_ACCESS_TOKEN }}
-          develop-branch: develop
-          release-branch: main
+          develop-branch: main
           version-files: src/common/configuration.go
```

```suggestion
          version-files: src/common/configuration.go README.md
```

can we add the readme file here because of:
 https://github.com/Adyen/adyen-go-api-library/blob/07009b3d421285f444664e666094ebabd2956c95/Readme.md?plain=1#L69

Or will it break and we should do it manually?

---

Good catch! We can (auto) update the readme as well, PR updated!